integrations/google-analytics/lib/index.js

```diff
@@ -228,6 +228,13 @@ GA.prototype.page = function(page) {
     title: pageTitle
   };
 
+  // Reset custom dimension which are previously set.
+  var customDimension = {};
+  each(opts.dimensions, function(prop) {
```

Comment:
Would be possible to make this a regular `for...loop`? Also, do you think it makes sense to use `delete customDimension[prop]`?